conbench/templates/c-benchmark-results-for-case.html

```diff
@@ -137,53 +143,71 @@ <h4 class="mb-3">
             sorted: 0,
             show: true,
             spanGaps: false,
-            stroke: 'rgb(87, 125, 134)',
-            width: 0.5,
+            stroke: "#812570", // dark magenta, main line color
+            width: 0.6,
             // I have been taking greenish colors from this palette:
             // https://colorhunt.co/palette/b9eddd87cbb9569daa577d86 We can
             // think about greenifying some graphs, and reddifying others,
             // based on their properties.
-            fill: "rgba(135, 203, 185, 0.4)",
+            fill: "#38BDD122", // skyblue, transp
             //fill: "rgba(255, 0, 0, 0.3)",
             //dash: [10, 5],
             scale: "y",
             points: {
-                // Without `show: true`, points do not always show.
+              // Without `show: true`, points do not always show.
               show: true,
               size: 5,
               fill: 'rgba(87, 125, 134, 0.7)'
             }
           }
         ],
         axes: [
-          {},
+          {
+            label: "benchmark time",
+            labelFont: "bold 12px 'Roboto Mono'",
+            font: "11px 'Roboto Mono'",
+            stroke: "#812570", // dark magenta
+            // 24 hour format on time axis ticks, remove am/pm, etc.
+            // Copied the default value from docs, then edited.
+            values: [
+              [3600 * 24 * 365,   "{YYYY}",         null,                            null,    null,                      null,    null,              null,        1],
+              [3600 * 24 * 28,    "{MMM}",          "\n{YYYY}",                      null,    null,                      null,    null,              null,        1],
+              [3600 * 24,         "{MM}/{DD}",        "\n{YYYY}",                      null,    null,                      null,    null,              null,        1],
+              [3600,              "{HH}",        "\n{YYYY}-{MM}-{DD}",                null,    "\n{MM}-{DD}",               null,    null,              null,        1],
+              [60,                "{HH}:{mm}",   "\n{YYYY}-{MM}-{DD}",                null,    "\n{MM}-{DD}",               null,    null,              null,        1],
+              [1,                 ":{ss}",          "\n{YYYY}-{MM}-{DD} {HH}:{mm}",   null,    "\n{MM}-{DD} {HH}:{mm}",  null,    "\n{HH}:{mm}",  null,        1],
+              [0.001,             ":{ss}.{fff}",    "\n{YYYY}-{MM}-{DD} {HH}:{mm}",   null,    "\n{MM}-{DD} {HH}:{mm}",  null,    "\n{HH}:{mm}",  null,        1],
+            ],
+          },
           {
             label: "{{ y_unit_for_all_plots }}",
+            labelFont: "bold 12px 'Roboto Mono'",
+            font: "11px  'Roboto Mono'",
+            stroke: "#812570", // dark magenta
```

Comment:
I think that Roboto Mono is/can be a phenomenal choice for the 'tiny things', but I rightfully didn't spend time on this.

Some good resources about fonts in data presentation anyway:

https://medium.com/nightingale/choosing-a-font-for-your-data-visualization-2ed37afea637

https://blog.datawrapper.de/fonts-for-data-visualization/

https://graphicdesign.stackexchange.com/questions/9246/what-are-great-fonts-for-information-heavy-data-tables

https://www.untitledui.com/blog/best-free-fonts

https://xdgov.github.io/data-design-standards/components/typography